packages/hoppscotch-backend/src/user/user.service.ts

```diff
@@ -316,6 +320,17 @@ export class UserService {
     }
   }
 
+  /**
+   * Update user's lastLoggedOn timestamp
```

Comment:
Can you rewrite this function to handle errors too, it currently is not a unit function.